src/components/snapshots/takeSnapshot.ts

```diff
@@ -58,6 +58,11 @@ const takeSnapshot: SnapshotOperation = async ({
       execSync(
         `pg_dump -U postgres tmf_app_manager --format=custom -f ${newSnapshotFolder}/database.dump`
       )
+      // This plain-text .sql script is NOT used for re-import, but could be
+      // useful for debugging when dealing with troublesome snapshots
+      execSync(
+        `pg_dump -U postgres tmf_app_manager --format=plain --inserts -f ${newSnapshotFolder}/database.sql`
+      )
```

Comment:
I just added this as I think it might be useful for debugging. We're still getting that weird error on re-import where the snapshot loading seems to fail every *2nd* time, and I don't know why. But being able to incrementally run parts of this .sql script will help figure it out I think.

We can maybe remove this again once it's 100% stable, but then again it doesn't really hurt to leave it there.

Reply:
> We're still getting that weird error on re-import where the snapshot loading seems to fail every 2nd time, and I don't know why.

This is fixed now, FYI